src/libtiled/wangset.h

```diff
@@ -0,0 +1,170 @@
+/*
+ * wangset.h
+ * Copyright 2017, Benjamin Trotter <bdtrotte@ucsc.edu>
+ * This file is part of libtiled.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ *    1. Redistributions of source code must retain the above copyright notice,
+ *       this list of conditions and the following disclaimer.
+ *
+ *    2. Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the
+ *       documentation and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE CONTRIBUTORS ``AS IS'' AND ANY EXPRESS OR
+ * IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO
+ * EVENT SHALL THE CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+ * WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+ * OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+ * ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#pragma once
+
+#include "tile.h"
+#include "tileset.h"
+
+#include <QHash>
+#include <QMultiHash>
+#include <QString>
+#include <QList>
+
+namespace Tiled {
+
+class WangId
+{
+public:
+    WangId(unsigned id) : mId(id) {}
+    WangId() : mId(0) {}
+
+    operator unsigned() const { return mId; }
+    inline void setId(unsigned id) { mId = id; }
+
+    /**
+     * @brief getColor returns the color of a desired edge/corner of a given wang id
+     * @param index index 0-3 with zero being the top right, and 3 left top
+     * @return
+     */
+    int edgeColor(int index) const;
+    int cornerColor(int index) const;
+
+    /**
+     * @brief rotateWangId rotates the wang id 90 * rotations degrees ccw
```

Comment:
Docs seem to mention old function name? I'd suggest to leave out the `@brief` and the function name.